manifests/deis-router-rc.yaml

```diff
@@ -23,3 +23,7 @@ spec:
         ports:
         - containerPort: 80
           hostPort: 80
+        - containerPort: 443
+          hostPort: 443
+        - containerPort: 2222
```

Comment:
Builder would have 2222 open within its container, and a service resource provides a VIP:2222 that maps to that.  That's actually how all the components work.  Router is an exception to that.  Since router is meant to be the single point of ingress for the Deis platform, router needs port mappings firectly from the host into its container.  This makes router unique among the components.... it will directly answer traffic received by the host on these ports.  This would allow, for instance, a load balancer outside the cluster to front (potentially) _all_ the nodes, but through a health check, determine which ones can actualky receive traffic.  This does differ a bit from how k8s _wants_ to do things.  It would rather, for instance that we expose a service of `type: LoadBalancer` for the router, which would cause k8s to use the applicable "cloud provider" (which is really an _adapter_ to your underlying infrastructure provider) to create the load balancer for you.  Ultimately, we may want to do that still, but on Vagrant or bare metal, for instance, the bits that manage a load balancer external to the cluster are a no op... which forces our hand.  We have to employ the host-> container port mapping strategy seen here to provide a point of ingress that's guaranteed to work universally.